### ISSUE TITLE
[Linux] userdata directory not in location expected by ice_engine.py

### ISSUE THREAD

**Issue by Awjin** [scottrice/Ice#306]
I ran into the following error while trying to run `python src/ice.py`:

````
Traceback (most recent call last):
  File "src/ice.py", line 11, in <module>
    runner.run(sys.argv)
  File "/home/USERNAME/Ice/src/ice/runners/command_line_runner.py", line 27, in run
    engine = IceEngine(options)
  File "/home/USERNAME/Ice/src/ice/runners/ice_engine.py", line 60, in __init__
    self.users = self.steam.local_users()
  File "/usr/local/lib/python2.7/dist-packages/pysteam-0.1.3-py2.7.egg/pysteam/steam.py", line 78, in local_users
    for entry in os.listdir(userdata_dir):
OSError: [Errno 2] No such file or directory: '/home/USERNAME/.local/share/Steam/userdata'
````

Went digging, found that my **userdata directory is actually in `~/.steam/steam/userdata`**.

I saw this comment in `src/ice/runners/ice_engine.py` line 60:

````
    # TODO: Query the list of users some other way
    self.users = self.steam.local_users()
````

So, I tinkered a bit with `steam.py` (in `dist-packages`) line 58:

<pre>
        elif _is_linux():
            return os.path.join(os.path.expanduser("~"),
                                #".local",
                                #"share",
                                #"Steam",
                                #"userdata"
                                <b>".steam",
                                "steam",
                                "userdata"</b>
            )
</pre>

This worked. I was able to run `ice.py`.

**In summary**, there might be something different now in linux Steam, therefore the userdata folder is located elsewhere, breaking the install.

Cheers,
Awj
